packages/test-runner/src/test.ts

```diff
@@ -269,6 +269,12 @@ export class Suite extends Runnable {
     return false;
   }
 
+  * _allTests(): Iterable<Test> {
```

Comment:
A big array of tests is really cheap to create, let's not use generators for this.